python/setup.py

```diff
@@ -91,8 +56,7 @@ def locate_nvgraph():
               sources=CYTHON_FILES,
               include_dirs=[NUMPY_INCLUDE,
                             CUDF_INCLUDE,
-                            NVGRAPH['include'],
-                            CUDA['include'],
+                            #NVGRAPH['include'],
```

Comment:
Is this commented on purpose, and if so should we just delete this?

Reply:
can be deleted but could be required by upcoming algorithms added to cugraph